pandas/core/resample.py

```diff
@@ -1358,7 +1358,7 @@ def ohlc(self):
         return self._downsample("ohlc")
 
     @final
-    @doc(SeriesGroupBy.nunique)
+    @doc(SeriesGroupBy.nunique, extra_params="")
```

Comment:
Can you give this a distinct docstring (e.g. copy-paste the SeriesGroupBy.nunique docstring here) and avoid using doc?

Reply:
Sounds good. Since they're now their own distinct docstrings, I separated the examples too (added see alsos to each other instead)